debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlStreamingChangeEventSource.java

```diff
@@ -492,7 +493,7 @@ protected void handleQueryEvent(MySqlPartition partition, MySqlOffsetContext off
             return;
         }
 
-        String upperCasedStatementBegin = Strings.getBegin(sql, 7).toUpperCase();
+        String upperCasedStatementBegin = Strings.getBegin(removeSetStatement(sql), 7).toUpperCase();
```

Comment:
@jpechane Any idea of what this magic number means here?

Reply:
@mfvitale Unforutnately I don't recall that sorry :-(